cli/cmd/vuln_container.go

```diff
@@ -246,6 +255,18 @@ To request an on-demand vulnerability scan:
 
     lacework vulnerability container scan <registry> <repository> <tag|digest>`,
 		Args: cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if vulCmdState.Csv {
+				cli.EnableCSVOutput()
+
+				// If --details or --packages is not passed, csv outputs nothing; defaulting to --details
```

Comment:
why? if I pass `--csv` with or without those two flags, I think we should just output CSV.

This is the same behavior if someone passes `--pdf`, we just output it. (meaning it has more priority)

Reply:
The challenge with that is the outputs from these commands essentially output 3 tables.  Typically a host summary, the details summary, and the data relevant to CSV output is only generated if you pass `--details` or `--packages`. 

So we can either add these implicit flags as I did here, or we can build a totally different 'path' in the code for CSV output.  Let's discuss this further.